distro/pom.xml

```diff
@@ -22,430 +22,36 @@
                     <value>true</value>
                 </property>
             </activation>
-            <dependencies>
-                <dependency>
-                    <groupId>org.openmrs.module</groupId>
-                    <artifactId>mirebalais-omod</artifactId>
-                    <version>${project.parent.version}</version>
-                </dependency>
-
-                <!-- All the -omod dependencies for modules we depend on, including transitive dependencies since we
-                     aren't guaranteed that OpenMRS modules declare those in their poms. -->
```

Comment:
I forget exactly what this means/was for, but you determine we no longer need all of this?

Reply:
Agree this needs to be coordinated with debian package changes.  This was just step 1.  Yes, I'm assuming this isn't required.  Ultimately, as long as our expected modules are pulled into the distro, that's all we want.  And since the SDK is already what we use in our dev environments, and this is just saying "use same thing the SDK uses", I think we're good.